config/core/rootfs-configs.yaml

```diff
@@ -219,6 +219,8 @@ rootfs_configs:
       - perl-modules-5.28
       - procps
       - publicsuffix
+      - python3-minimal
+      - python3-unittest2
```

Comment:
You'll have to re-apply these change when bullseye-kselftest has been merged, maybe today or Monday

Reply:
Sure, no problem.

Reply:
Actually I just merged it, so please rebase anytime :)

Reply:
Hm, now that I think about it, since this PR both changes the rootfs and adds the test, shouldn't I wait for the kselftest test definition to be updated to use the new `buster-kselftest` rootfs before I rebase? Or I could also split this PR between the rootfs and the test part if you prefer.

Reply:
OK, the bullseye rootfs seems to be in place now, so I've rebased and force-pushed.